pages/common/qutebrowser.md

```diff
@@ -0,0 +1,24 @@
+# qutebrowser
+
+> A keyboard-driven, vim-like browser based on PyQt5.
+> More information: <https://qutebrowser.org/>.
+
+- Open qutebrowser with a specified (non-default) storage directory:
+
+`qutebrowser --basedir {{base_directory}}`
+
+- Open a qutebrowser instance with temporary settings:
+
+`qutebrowser --set {{option}} {{value}}`
+
+- Restore a named session of a qutebrowser instance:
+
+`qutebrowser --restore {{session_name}}`
```

Comment:
Same here.  How can one access a list of valid session names?

Reply:
As far as I'm aware of, there is not an argument to list (at time of writing, might open a PR there..) saved named sessions. These can be found in the browser itself (`:session-load {tab}`) via the session loading command.